src/main/java/seedu/address/logic/commands/customer/CustomerEditCommand.java

```diff
@@ -100,7 +108,11 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(MESSAGE_DUPLICATE_PERSON);
         } else {
             model.setCustomer(customerToEdit, editedCustomer);
+
+            // update delivery associated with the edited customer
+            updateDelivery(model, editedCustomer);
             model.updateFilteredCustomerList(PREDICATE_SHOW_ALL_CUSTOMERS);
```

Comment:
May want to merge master into current branch to show all filtered customer list.